ocf_datapipes/training/example/README.md

```diff
@@ -4,7 +4,7 @@ These are example datapipes using the `ocf_datapipes` library
 
 ## Simple PV
 
-`simple_pv.py` has a training pipeline for just loading PV data.
+[`simple_pv.py`](simple_pv.py) has a training pipeline for just loading PV data.
```

Comment:
Removed one of the example diagrams and file mentioned that is no longer in the repo and then just reordered examples from simplest to most complex and added links, more clear to see that [here](https://github.com/openclimatefix/ocf_datapipes/blob/087d022c063eed28622544d649e8db98aa488d58/ocf_datapipes/training/example/README.md), just the order